src/app/pages/common/info-dialog/info-dialog.component.scss

```diff
@@ -1,5 +1,12 @@
 .info-panel {
     max-height: 115px !important;
```

Comment:
Set the max-height to `80vh !important`

Reply:
The parent block `.mat-dialog-content` has `max-height: 65vh`, it may be an issue whenever the output is very long.